github/repos.go

```diff
@@ -1321,3 +1321,28 @@ func (s *RepositoriesService) Transfer(ctx context.Context, owner, repo string,
 
 	return r, resp, nil
 }
+
+// RepositoryDispatch represents a repository_dispatch event.
+type RepositoryDispatch struct {
+	EventType     string            `json:"event_type"`
+	ClientPayload map[string]string `json:"client_payload"`
+}
+
+// CreateRepoDispatch triggers a webhook event called repository_dispatch
+// when you want activity that happens outside of GitHub to trigger a
+// GitHub Actions workflow or GitHub App webhook.
+//
+// GitHub API docs: https://developer.github.com/v3/repos/#create-a-repository-dispatch-event
+func (s *RepositoriesService) CreateRepoDispatch(ctx context.Context, owner, repo, repoDispatch RepositoryDispatch) (*Response, error) {
```

Comment:
This is missing a `string` type for `owner, repo`, and I think we would prefer a pointer to the last parameter, although that is debatable. 
`func (s *RepositoriesService) CreateRepoDispatch(ctx context.Context, owner, repo string, repoDispatch *RepositoryDispatch) (*Response, error) {`